girder/api/v1/notification.py

```diff
@@ -78,6 +77,9 @@ def streamGen():
                     wait = MIN_POLL_INTERVAL
                     start = time.time()
                     yield sseMessage(event)
+                if (cherrypy.engine.state != cherrypy.engine.states.STARTED or
+                        time.time() - start > timeout):
+                    break
```

Comment:
Should the conditional about `cherrypy.engine.states.STARTED` still be in the `while` condition?  That is, if cherrypy isn't started, do we ever want to yield an sseMessage?